src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+        protected String description;
+        protected boolean isDone;
+
+        public Task(String description) {
+            this.description = description;
+            this.isDone = false;
+        }
+
+        public String getStatusIcon() {
+
+            return (isDone ? "X" : " "); // mark done task with X
+        }
+
+        public void markAsDone()
+        {
```

Comment:
Should this bracket be moved to the previous line to follow the K&R style brackets?